### PR TITLE
API provides functionality to display article

### PR DESCRIPTION
**Pull request by maxarvid**
Link to PT: https://www.pivotaltracker.com/story/show/179938239

## User Story

```
As a news API
In order to give users the full story
I would like to return a full article when requested
```

## Tasks

- Create `show` method in `articles_controller.rb`
- Set up `show` route
- Add `body` attr. to article table
- Update factory bot, model validations, and create article spec tests for `body` attr.
- Include rescue for record not found in application_controller

## Screenshot

Screenshot 2021-10-15 at 15 43 21

Screenshot 2021-10-15 at 15 44 15

Screenshot 2021-10-15 at 15 44 29

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,5 @@
 
 # Test Coverage
 /coverage
+
+.rufo
```
Comment (offcast-dev) [inline .gitignore:26]: its a formatter right? Is that switch intended? Do every one have the same formatter and settings?

**Review by dorianbuck — COMMENTED** [review]
👍🏽

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,2 +1,9 @@
 class ApplicationController < ActionController::API
+  rescue_from ActiveRecord::RecordNotFound, with: :record_not_found_error
+
+  private
+
+  def record_not_found_error
+    render json: { errors: 'Your request could not be processed.' }, status: 422
+  end
```
Comment (offcast-dev) [inline app/controllers/application_controller.rb:8]: very good

**Review by offcast-dev — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (johnnysee) [reply .gitignore:26]: Everyone is using Solargraph except me. I have errors using Solargraph.

**Review by offcast-dev — APPROVED** [review]
Very good job!

**Pull request merged**